### PR TITLE
Wiki documentation for repository filters

### PR DESCRIPTION
**Pull request by andpopov**
Part of: #1254

### CODE REVIEW

**Comment by andpopov** [comment]
@olenagerasimova, @dgarus, @ChGen Please review

File: .wiki/Configuration-Repository.md [hunk 1]
```diff
@@ -63,3 +63,76 @@ repo:
...
+- ``include`` list of patterns of allowed resources.
+- ``exclude`` list of patterns of forbidden resources.
+```
+filters:,
+  include:
+    ...
+  exclude:
+    ...
+```
+Each http-request to repository resource are controlled by filters.
```
Comment (olenagerasimova) [inline .wiki/Configuration-Repository.md:81]: are -> is ("each request" is single)

File: .wiki/Configuration-Repository.md [hunk 2]
```diff
@@ -63,3 +63,76 @@ repo:
...
+The following rules are used to get access to repository resource:
+- **Repository resource is allowed** if it matches at least one of patterns in the ``include`` list and does not match any of patterns in the ``exclude`` list.
+- **Repository resource is forbidden** if it matches at least one of patterns in the ``exclude`` list or both list of patterns ``include`` and ``exclude`` are empty.
+
+Artipie provides out-of-the-box following pattern matching types:
+- **glob** patters specify sets of resource locations with wildcard characters. Description of platform specific glob syntax is [here](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/FileSystem.html#getPathMatcher(java.lang.String))
```
Comment (olenagerasimova) [inline .wiki/Configuration-Repository.md:87]: patters -> patterNs

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@andpopov thank, please, check my comments. Also, why are you using pair of two quotes `` for inline code? In wiki github markdown works fine and we use one quote for inline code in all wiki pages. I'm not sure how this pair of two quotes will look like after publishing...

**Comment by andpopov** [comment]
@olenagerasimova Thanks a lot, I have corrected

**Review by olenagerasimova — APPROVED** [review]
thanks

**Review by dgarus — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**